.gitignore

```diff
@@ -22,3 +22,4 @@ erl_crash.dump
 # Ignore package tarball (built via "mix hex.build").
 surface-*.tar
 
+.elixir_ls
```

Comment:
If this is something that you want to ignore but is relevant only to your setup you can use `.git/info/exclude`. Or do you think this should be here for everyone?

Reply:
One nice alternative is to add `.elixir_ls` to `~/.gitignore_global`. That's what I do so I don't have to set it up in every project.

Reply:
Thanks, I didn't know.